plugin-modernizer-core/src/main/java/io/jenkins/tools/pluginmodernizer/core/impl/PluginModernizer.java

```diff
@@ -204,9 +204,15 @@ private void process(Plugin plugin) {
             if (config.isRemoveForks()) {
                 plugin.deleteFork(ghService);
             }
-            plugin.fork(ghService);
-            plugin.sync(ghService);
             plugin.fetch(ghService);
+            plugin.commit(ghService);
+            if (plugin.hasChangesPushed()) {
```

Comment:
Always false if we don't push changes before.

Where is the part we fetch always the original repo?

Where is the part that change the remote  before pushing?

Randomly changing the order without proving it improve the situation, will be an immediate closing of the PR